doc/HOWTO_forks.md

```diff
@@ -41,3 +41,19 @@ the commits in your branch so you only need to bring one commit.
 git fetch nu
 git cherry-pick XXXXX
 ```
+
+# Deploy
+From the root of your fork:
+
+- `bundle exec cap [staging_name] deploy`
+
+Find `staging_name` by the name of the `.rb` files in `config/deploy`
+
+To confirm deploy was successful:
+
+- `ssh nucore@nucore.rrc.uic.edu`
+- `ls -al`
+- `cd nucore.rrc.uic.edu/current`
```

Comment:
you can also check out what `current` symlinks to. It should symlink to `sitedomain.edu/releases/20170814XXXXXX`, and the timestamp of the release should look like it's very recent. That doesn't really tell you if the release was successful, though.